base/initdefs.jl

```diff
@@ -170,6 +170,11 @@ const LOAD_PATH = copy(DEFAULT_LOAD_PATH)
 # HOME_PROJECT is no longer used, here just to avoid breaking things
 const HOME_PROJECT = Ref{Union{String,Nothing}}(nothing)
 const ACTIVE_PROJECT = Ref{Union{String,Nothing}}(nothing)
+## Watchers for when the active project changes (e.g., Revise)
+# Each should be a thunk, i.e., `f()`. To determine the current active project,
+# the thunk can query `Base.active_project()`.
+# Modify it only via `Base.set_active_project(proj)`
```

Comment:
Should this comment be on the line `const ACTIVE_PROJECT = Ref{Union{String,Nothing}}(nothing)`